src/services/storageParser/extractAboutUsText.ts

```diff
@@ -1,47 +1,55 @@
 
+import { url } from 'inspector';
 import openAi from '../../lib/openai.ts';
 
-async function validate() {
-    // const askAboutUsInfoPrompt = `` +
-    // `${textString}` +
-    // `\n\n` +
-    // `O texto acima possui informação sobre o site ${url}? Utilize apenas o texto acima como informação. Responda "SIM" ou "NAO".`
+async function validate(textString, url) {
+    const reduceTextString = textString.slice(0, 3000)
+    const askAboutUsInfoPrompt = `` +
+    `${reduceTextString}` +
+    `\n\n` +
+    `O texto acima possui informação sobre o site ${url}? Utilize apenas o texto acima como informação. Responda "SIM" ou "NAO".`
 
-    // const result = await openAi.createCompletion(askAboutUsInfoPrompt);
-    // console.log(result);
+    const result = await openAi.createCompletion(askAboutUsInfoPrompt);
+    return result
 }
 
-async function extract () {
-    // const extractAboutUsInfoPrompt = `` +
-    // `${textString}` +
-    // `\n\n` +
-    // `Apenas usando o texto acima, extraia a informação sobre o site ${url}.`
+async function extract (textString, url) {
+    const reduceTextString = textString.slice(0, 3000)
+    const extractAboutUsInfoPrompt = `` +
+    `${reduceTextString}` +
+    `\n\n` +
+    `Apenas usando o texto acima, extraia a informação sobre o site ${url}.`
 
-    // const result = await openAi.createCompletion(extractAboutUsInfoPrompt);
+    const result = await openAi.createCompletion(extractAboutUsInfoPrompt);
+    return result
 }
 
 
 async function extractAboutUsText(cleanedStorage) {
-    const firstItem = cleanedStorage[0]
-
-    const hasAboutUsInfo = await validate(firstItem.text)
-    // - Se sim, perguntar para extrair a informação sobre o site
-    // - Se não, retorna uma string vazia
-    const aboutUsInfo = await extract(firstItem.text)
-
     console.log('rodando extractAboutUsText')
 
-    // return cleanedStorage.map((siteText) => {
-    //     const url = siteText.url
-    //     const text = siteText.text
-    
-    //     // TODO
+    const extractedAboutUsText = []
+    for (const siteText of cleanedStorage) {
+        const url = siteText.url
+        const text = siteText.text
+        console.log('Verificando a url: ', url)
     
-    //     return {
-    //         url,
-    //         text,
-    //     }
-    // })
+        const hasAboutUsInfo = await validate(text, url)
+        if (hasAboutUsInfo === 'SIM') {
+            const aboutUsInfo = await extract(text, url)
+        
+            extractedAboutUsText.push({
+                url,
+                text: aboutUsInfo,
+            })
+        }
+        else {
+            extractedAboutUsText.push({
+                url,
+                text: ''})
+        }
+    }
+    return extractedAboutUsText
```

Comment:
Alguns ajustes de formatação:

```suggestion
        const hasAboutUsInfo = await validate(text, url);
        
        if (hasAboutUsInfo === 'SIM') {
            const aboutUsInfo = await extract(text, url);
        
            extractedAboutUsText.push({
                url,
                text: aboutUsInfo,
            });
        } else {
            extractedAboutUsText.push({
                url,
                text: ''
            });
        }
    }
    
    return extractedAboutUsText;
```